jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
Should work, hopefully it should give any side effects. But with this change, you need to rely on the try/except just below when doing the Jira query. From programming practice point of view, it's a bit hacky. You shouldn't write the code in such a way that the expect case is part of the normal flow.